cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/FileExporter.java

```diff
@@ -115,19 +107,9 @@ String getRaoParametersDestinationPath(String basePath, ProcessType processType,
 
     RaoParameters getRaoParameters(List<String> remedialActionsAppliedInPreviousStep) {
         RaoParameters raoParameters = loadRaoParameters();
-        try (InputStream is = new FileInputStream(combinedRasFilePath)) {
-            ObjectMapper objectMapper = new ObjectMapper();
-            List<List<String>> combinedRas = objectMapper.readValue(is.readAllBytes(), List.class);
-            if (!remedialActionsAppliedInPreviousStep.isEmpty()) {
-                combinedRas.add(remedialActionsAppliedInPreviousStep);
-            }
-            SearchTreeRaoParameters searchTreeRaoParameters = raoParameters.getExtension(SearchTreeRaoParameters.class);
-            if (searchTreeRaoParameters != null) {
-                searchTreeRaoParameters.setNetworkActionIdCombinations(combinedRas);
-                searchTreeRaoParameters.setMaxCurativeRaPerTso(MAX_CURATIVE_RA_PER_TSO);
-            }
-        } catch (IOException e) {
-            throw new CseDataException(String.format("Impossible to read combined RAs file: %s", combinedRasFilePath));
+        List<List<String>> combinedRas = raoParameters.getTopoOptimizationParameters().getPredefinedCombinations();
+        if (!remedialActionsAppliedInPreviousStep.isEmpty()) {
+            combinedRas.add(remedialActionsAppliedInPreviousStep);
```

Comment:
the combinedRas was not set in the raoParameters. (you just added remedialActionsAppliedInPreviousStep in combinedRas and this list was not used thereafter).